ipatests/test_integration/test_external_ca.py

```diff
@@ -279,3 +279,77 @@ def test_install_external_ca(self):
         # Install new cert
         self.master.run_command([paths.IPA_CACERT_MANAGE, 'install',
                                  root_ca_fname])
+
+
+class TestMultipleExternalCA(IntegrationTest):
+    """setup ext-ca1 install ipa-server with ca1
+     setup ext-ca2 renew ipa-server with ext-ca2
+     check the difference in certificate"""
+
+    def test_master_install_ca1(self):
+
+        install_server_external_ca_step1(self.master)
+        # Sign CA, transport it to the host and get ipa a root ca paths.
+
+        test_dir1 = os.path.join(self.master.config.test_dir, 'CA1')
```

Comment:
I would recommend to create temporary directory here e.g.:
`tempfile.mkdtemp(prefix="extCA1-")`

This way you do not need to do cleanup (which is missing here) later.

Applies to all other directories.